cloud_images/centos7/create_dcos_ami.sh

```diff
@@ -5,7 +5,7 @@ set -o errexit -o nounset -o pipefail
 export AWS_PROFILE=${AWS_PROFILE:-"development"}
 
 # Base CentOS 7 AMI and region
-export SOURCE_AMI=${SOURCE_AMI:-"ami-31a8ca51"}
+export SOURCE_AMI=${SOURCE_AMI:-"ami-a9b24bd1"}
```

Comment:
8 character fix after 6 hours with 2 people. :)

Reply:
Indeed. AFAICT the root cause for this problem is not fixed. How can we make sure the exact same thing does not happen again when the AMI is updated @orsenthil ?

Reply:
You're asking how to remove the potential for human error? Or something else?